dev-python/svg-path/svg-path-3.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+MY_PN="svg.path"
```

Comment:
Make it into `MY_P`, and then use on right hand side of `->` in SRC_URI.

Reply:
That just makes it more confusing, I end up with a non-standard named distfile which on top of everything contains a period as a non-extension delimiter.

Reply:
If project is named 'svg.path', then that's how distfile should be named. Otherwise, you end up with directory name that doesn't match distfile name which makes no sense. Also, 'extension' is Windows thinking. Forget about it.